addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -77,7 +77,9 @@ def _process_step_create_lead(self, discuss_channel):
             create_values = {"partner_id": self.env.user.partner_id.id}
         create_values.update(self._chatbot_crm_prepare_lead_values(
             discuss_channel, customer_values['description']))
-        return self.env["crm.lead"].create(create_values)
+        new_leads = self.env["crm.lead"].create(create_values)
+        new_leads._assign_userless_lead_in_team()
```

Comment:
Idea: why not adding a "source" parameter, allowing to tune the logged message ? Like ``This new lead created by livechat discussion was automatically assigned to team leader Raoulette Mobylette.`` if source == "livechat". Would give some hints on lead source in a quick way :) .